auto_submit/lib/service/bigquery.dart

```diff
@@ -0,0 +1,462 @@
+// Copyright 2022 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:async';
+
+import 'package:auto_submit/exception/bigquery_exception.dart';
+import 'package:auto_submit/model/big_query_pull_request_record.dart';
+import 'package:auto_submit/model/big_query_revert_request_record.dart';
+import 'package:googleapis/bigquery/v2.dart';
+import 'package:http/http.dart';
+
+import 'access_client_provider.dart';
+
+const String selectRevertRequestDml = r'''
+SELECT organization,
```

Comment:
At this point, should we just say `SELECT *`?

Reply:
The only reason I did this is because I was making sure that we return the fields in the correct order we expect them to be since the bigquery implementation is lacking. But I can change this to * if that is better.

Reply:
Scratch that I am going to leave this as is. Since parsing of the row is index based this gives clarity to the data returned as we cannot specify data selection by field.